packages/wasm-dpp/src/errors/consensus/basic/identity/identity_asset_lock_proof_locked_transaction_mismatch_error.rs

```diff
@@ -28,7 +29,10 @@ impl IdentityAssetLockProofLockedTransactionMismatchErrorWasm {
     #[wasm_bindgen(js_name=getAssetLockTransactionId)]
     pub fn asset_lock_transaction_id(&self) -> Buffer {
         let tx_id = self.inner.asset_lock_transaction_id();
-        Buffer::from_bytes(tx_id.to_vec().as_ref())
+        let mut hash_bytes = tx_id.as_hash().into_inner();
+        hash_bytes.reverse();
+
+        Buffer::from_bytes(hash_bytes.as_slice())
```

Comment:
Passing `&hash_bytes` didn't work? What's the specific reason to call `as_slice` here?

Reply:
Fixed